README.md

```diff
@@ -10,20 +10,38 @@
 Repository to describe, develop, document and test the Connectivity Insights API family
 
 ## Scope
-* Service APIs for “Connectivity Insights” (see APIBacklog.md)  
-* It provides the customer with the ability to:  
+* Service APIs for “Connectivity Insights” (see APIBacklog.md)
+* It provides the customer with the ability to:
   * define intents in the form of policy thresholds for QoS metrics against the device and the application service. The API service will alert the consumers if and when the policy has breached.
-  * NOTE: The scope of this API family should be limited (at least at a first stage) to 4G and 5G.  
-* Describe, develop, document and test the APIs (with 1-2 Telcos)  
+  * NOTE: The scope of this API family should be limited (at least at a first stage) to 4G and 5G.
+* Describe, develop, document and test the APIs (with 1-2 Telcos)
 * Started: November 2023
-* Location: virtually  
+* Location: virtually
```

Comment:
This line could be deleted, as it would belong to the meetings, not to the scope.
```suggestion
```